### PR TITLE
Fix broken hyperlinks in documentation [skip ci]

### LINKED ISSUE #4725: [DOC]broken links in guide doc
broken links:
tunning-guide.md line 338 is available in the <a href="co 
download.md line 581 [Spark shuffle for GPUs](get-started/getting-started-on-prem.md#enabling-rapids 
getting-started-gcp.md and some notebook https://rapidsai.github.io/demos/datasets/mortgage-data
getting-started-workload-qualification.md line 34 (../spark-profiling-tool.md) to analyze them
rapids-udfs.md https://docs.rapids.ai/api/cudf-java/stable
getting-started-workload-qualification.md line 91 and line 133 ../configs.md

### PR DESCRIPTION
**Pull request by nvliyuan**
fix broken links in branch2202
fix #4725

### CODE REVIEW

**Comment by nvliyuan** [comment]
derived from #4743

File: docs/get-started/getting-started-gcp.md [hunk 1]
```diff
@@ -86,8 +86,8 @@ If you'd like to further accelerate init time to 4-5 minutes, create a custom Da
 To use notebooks with a Dataproc cluster, click on the cluster name under the Dataproc cluster tab
 and navigate to the "Web Interfaces" tab.  Under "Web Interfaces", click on the JupyterLab or
 Jupyter link to start to use sample [Mortgage ETL on GPU Jupyter
-Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb) to process full 17 years [Mortgage
-data](https://rapidsai.github.io/demos/datasets/mortgage-data).
+Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb) to process full 17 years
```
Comment (jlowe) [inline docs/get-started/getting-started-gcp.md:89]: [Same comment](https://github.com/NVIDIA/spark-rapids/pull/4743#discussion_r803743695) I posted on the previous PR.  This link is split across lines, which the other PR said it was going to address.

Comment (nvliyuan) [reply docs/get-started/getting-started-gcp.md]: thanks, updated

**Review by jlowe — APPROVED** [review]

**Comment by jlowe** [comment]
build

**Pull request merged**